test/teslamate_web/live/geofence_live_test.exs

```diff
@@ -622,19 +622,19 @@ defmodule TeslaMateWeb.GeoFenceLiveTest do
                  geofence_id: id,
                  charge_energy_added: decimal(4.57),
                  charge_energy_used: nil,
-                 cost: decimal(6.5100)
+                 cost: decimal(6.5081)
                },
                %ChargingProcess{
                  geofence_id: id,
                  charge_energy_added: decimal(11.82),
                  charge_energy_used: nil,
-                 cost: decimal("8.9000")
+                 cost: decimal("8.9006")
                },
                %ChargingProcess{
                  geofence_id: id,
                  charge_energy_added: decimal("52.10"),
                  charge_energy_used: nil,
-                 cost: decimal(22.1900)
+                 cost: decimal(22.1930)
```

Comment:
I found that these needed to be quoted strings, e.g.
```
cost: decimal("22.1930")
```
otherwise the trailing zero is lost, and the test fails.

Reply:
That looks to have resolved it.  Thank you.

Reply:
Would love some help fixing the tests in #1684.  If you could help with that too I would be very appreciative :)

Reply:
> Would love some help fixing the tests in #1684. If you could help with that too I would be very appreciative :)

I took a quick look, but no idea on those - sorry!